src/loaders/tiffPyramidLoader.js

```diff
@@ -60,6 +60,13 @@ export default class TiffPyramidLoader {
     return { data, width, height };
   }
 
+  getRasterSize({ z }) {
+    const image = this.channelPyramids[0][z];
+    const imageWidth = image.getWidth();
+    const imageHeight = image.getHeight();
+    return { imageWidth, imageHeight };
+  }
```

Comment:
We changed `getRaster` to return just `width` & `height`. Should we do the same here?

Reply:
Will change.